src/app/components/dashboard-apps-dock/no-apps-notification.html

```diff
@@ -0,0 +1,6 @@
+<div>
```

Comment:
Initially, I had to wrap all in tags for the template to be completely rendered, since there was a locale unwrapped it would not show up. Since now I have changed the structure, the <div> is not longer needed.